packages/core/nuxt-theme-module/theme/components/AppHeader.vue

```diff
@@ -65,21 +65,32 @@ export default {
 
     const accountIcon = computed(() => isAuthenticated.value ? 'profile_fill' : 'profile');
 
-    const onAccountClicked = () => {
-      isAuthenticated && isAuthenticated.value ? root.$router.push('/my-account') : toggleLoginModal();
+    const handleSidebarOpen = async () => {
+      await loadCart();
```

Comment:
IMHO this should work differently. We cant have logic that is required for sidebar to work outside of the sidebar.
It could be either:
- watcher in the sidebar that loads the cart when `isOpen` becomes true
- fn in `onMounted` hook and sidebar will become async and lazily loaded